examples/apiserver-cert.yaml

```diff
@@ -1,20 +1,20 @@
 apiVersion: "giantswarm.io/v1"
 kind: Certificate
 metadata:
-  name: "example-cert"
+  name: "uo91f-apiserver"
+  namespace: "default"
 spec:
-  clusterID: "cert-test"
-  clusterComponent: "kubernetes"
-  commonName: "api.cert-test.g8s.eu-west-1.aws.test.private.giantswarm.io"
+  clusterID: "uo91f"
+  clusterComponent: "apiserver"
```

Comment:
I would prefer `api` over `apiserver`. The scheme for the other certs is to use the `clusterComponent` as sub domain identifier. The only exception is here.